### PR TITLE
COMMERCE-12394

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-12394

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 1]
```diff
@@ -440,6 +440,106 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12394. Buyer user can view quantity selector Pop Erros for not allowed quantities."
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:443]: Popover errors for not allowed quantities

**Review by pietrobocale — CHANGES_REQUESTED** [review]
Review Complete.
Check the comments!

Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:forward

**Comment by Brunno0liveira** [comment]
ci:reopen

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:forward

**Pull request closed without merging**